src/app/application/[owner]/[repo]/[id]/page.tsx

```diff
@@ -34,7 +36,7 @@ const ApplicationDetailPage: React.FC<ComponentProps> = ({
     return (
       <div className="p-10">
         <div className="mb-10">
-          <AppInfoCard application={data} />
+          <AppInfoCard application={data} repo={repo} owner={allocators.find((e) => e.repo === repo)!.owner} />
```

Comment:
Why do we need here to find the owner if we already have it in the function params?